src/tab/TabGroup.ts

```diff
@@ -0,0 +1,241 @@
+import { html, css, TemplateResult, nothing } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { OmniElement } from '../core/OmniElement.js';
+import { TabHeader } from './TabHeader.js';
+
+/**
+ * Component that displays content in tabs.
+ *
+ * @import
+ * ```js
+ * import '@capitec/omni-components/tab';
+ * ```
+ *
+ * @example
+ * ```html
+ * <omni-tab-group>
+ *  <omni-tab header='Tab 1'>
+ *   <omni-label>Tab 1</omni-label>
+ *  </omni-tab>
+ *  <omni-tab header="Tab 2">
+ *   <omni-label>Tab 2</omni-label>
+ *  </omni-tab>
+ *  <omni-tab header="Tab 3">
+ *   <omni-label>Tab 3</omni-label>
+ *  </omni-tab>
+ * </omni-tab-group>
+ *```
+ *
+ * @element omni-tab-group
+ *
+ * Registry of all properties defined by the component.
+ *
+ * @fires {CustomEvent<{ previous: HTMLElement, selected: HTMLElement}>} tab-select - Dispatched when an omni-tab is selected.
+ *
+ * @slot - All omni-tab components that are managed by this component.
+ * @slot header - Optional omni-tab-header components associated with each omni-tab component.
+ *
+ * @cssprop --omni-tabs-tab-bar-width - Tabs tab bar width.
```

Comment:
```suggestion
 * @cssprop --omni-tab-group-tab-bar-width - Tabs tab bar width.
```

Appy, to all "-tabs" related names.

Reply:
Updated